### PR TITLE
feat: projects section + fixed footer

### PR DESCRIPTION
**Pull request by Tulun**
1. Projects sections w/ styling from Mon, Oct 17th.
2. updated footer from Mon, Oct 17th.

Screen Shot 2022-10-17 at 4 29 08 PM
Screen Shot 2022-10-17 at 4 29 19 PM

### CODE REVIEW

File: components/Projects/Projects.tsx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  align-items: center;
+`;
+const BigProject = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px;
+  isolation: isolate;
+  border: 1px solid #e7e7e7;
```
Comment (ryanwolhuter) [inline components/Projects/Projects.tsx:134]: nit: raw color

File: components/Projects/Projects.tsx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    > div {
+      path {
+        fill: var(--red-500);
+      }
+    }
+  }
+`;
+const BigProjectText = styled.h3`
+  display: none;
+  font-family: "Halyard Display";
```
Comment (ryanwolhuter) [inline components/Projects/Projects.tsx:155]: nit: raw font

File: components/Projects/Projects.tsx [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+  flex-wrap: wrap;
+`;
+
+const SmallProject = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  isolation: isolate;
+  border: 1px solid #e7e7e7;
```
Comment (ryanwolhuter) [inline components/Projects/Projects.tsx:203]: nit: raw color

File: components/Projects/Projects.tsx [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+`;
+
+const ProjectsBlurb = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ProjectsBlurbHeader = styled.h2`
+  font-family: "Halyard Display";
```
Comment (ryanwolhuter) [inline components/Projects/Projects.tsx:267]: nit: raw font

File: components/Projects/Projects.tsx [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+  line-height: 115%;
+  letter-spacing: -0.01em;
+
+  color: var(--black);
+  max-width: 366px;
+`;
+
+const ProjectsBlurbSubheader = styled.h3`
+  margin-top: 24px;
+  font-family: "Halyard Display";
```
Comment (ryanwolhuter) [inline components/Projects/Projects.tsx:280]: nit: raw font

File: components/VoteParticipation/VoteParticipation.tsx [hunk 6]
```diff
@@ -81,6 +87,28 @@ const Header = styled.div`
   font: var(--text-xxl);
   width: 921px;
   letter-spacing: -0.01em;
+  position: relative;
+  background: -webkit-radial-gradient(center, 50% 50%, red, black);
```
Comment (ryanwolhuter) [inline components/VoteParticipation/VoteParticipation.tsx:91]: I see this is wip so ignore if you're already gonna do this.

seems like you've used the standard css `red` and `black` here, make sure to change this to the brand color vars.

**Review by ryanwolhuter — APPROVED** [review]
LGTM — let me know if you want me to re-review the wip parts

**Review by daywiss — APPROVED** [review]
lgtm just address ryans comments

File: components/VoteParticipation/VoteParticipation.tsx (hunk 6 above)
Comment (Tulun) [reply components/VoteParticipation/VoteParticipation.tsx:91]: yeah this is WIP, going to move on and tackle this later.

**Pull request merged**